src/osm_configurator/view/toplevelframes/create_project_frame.py

```diff
@@ -242,7 +240,7 @@ def freeze(self):
 
     def unfreeze(self):
         """
-        If this method is called, the frame returns into its previous interactable state.
+        If this method is called, the frame returns into its previous intractable state.
```

Comment:
alrigthj just remember, all the intractable changes are WRONG!

Reply:
ok